src/Apps/Sell/Routes/PhotosRoute.tsx

```diff
@@ -96,11 +96,11 @@ export const PhotosRoute: React.FC<PhotosRouteProps> = props => {
               <RouterLink
                 to="https://help.artsy.net/s/article/How-to-Take-Photos-That-Sell"
                 target="_blank"
-                inline
+                rel="noopener noreferrer"
```

Comment:
👀

Reply:
[It's good practice to provide this when a link should be opened in a new tab with `target="_blank"`.
](https://elementor.com/blog/noopener-noreferrer/?utm_source=google&utm_medium=cpc&utm_campaign=13060922353&utm_term=&gad_source=1&gclid=CjwKCAjwyo60BhBiEiwAHmVLJaqOVv4UXG6IqKWUL0YcHK2OZZYmIUqOmdyTqhEXyazLZ7biCBs_rhoCIMUQAvD_BwE)